pages/_app.js

```diff
@@ -1,10 +1,22 @@
 import GlobalStyle from "../styles";
+import { useState } from "react";
+import { initialProjects } from "@/db/data";
 
 export default function App({ Component, pageProps }) {
+  const [projects, setProjects] = useState(initialProjects);
+
+  function deleteProject(id) {
+    setProjects(projects.filter((project) => project.id !== id));
+  }
+
```

Comment:
Just a very petty remark. 🤓
Prefixing event handlers with `handle`, like `handleDeleteProject`, is a common React convention. It clarifies the function's role and enhances readability. Consistency in this naming across your codebase can make your code more intuitive and maintainable -  so please also check the other handler functions.

Reply:
👍🏾